index.html

```diff
@@ -61,236 +61,7 @@ <h2 class="name">
           <li><img src="./image/angelist 1.png" alt="n" /></li>
         </ul>
       </div>
-    </section>
```

Comment:
I have observed the following issue to be core requirement that you should consider fixing: 
- [ ] The headline name, that is, your name _**Aime**_ should be wrapped with an `h1` tag to improve the page's SEO to its optimum. Please fix this.
- [ ] The navbar hamburger icon is **NOT** positioned well. It should be positioned in the red spot on the image below. See the image below to see how it is currently 👇🏾 
![image](https://user-images.githubusercontent.com/84156714/219090462-79451b08-b7ff-4f4d-b73c-359f727216fa.png)

- [ ] The social media Icons should be wrapped in an anchor tag, an `a` tag. This is important because users will use this link to contact you when they visit your portfolio site.
- [ ] There is a text "main" below the social media icons, this text should be there. Kindly remove it.
![image](https://user-images.githubusercontent.com/84156714/219091775-8a77ab21-a805-49d0-9eba-5b551b89576e.png)